xarray/core/options.py

```diff
@@ -292,7 +292,11 @@ def __exit__(self, type, value, traceback):
 
 def get_options():
     """
-    Get options for xarray.
+    Retrieves the current global option for xarray
```

Comment:
Retrieve the current global options for xarray.

Reply:
Thanks for the correction, I'll update it!